skyportal/models/localization.py

```diff
@@ -141,3 +146,31 @@ def flat(self):
             return healpy.reorder(result, 'NESTED', 'RING')
         else:
             return (self.flat_2d,)
+
+    tiles = relationship("LocalizationTile")
+
+
+class LocalizationTile(Base):
+    """This is a single tile within a skymap (as in the Localization table).
+    Each tile has an associated healpix id and probability density."""
+
+    localization_id = sa.Column(
+        sa.ForeignKey('localizations.id', ondelete="CASCADE"),
+        nullable=False,
+        doc='localization ID',
+    )
+
+    localization = relationship(
+        "Localization",
+        foreign_keys=localization_id,
+        doc="The Localization that this tile belongs to",
+        passive_deletes=True,
+    )
```

Comment:
Again, you can safely delete the relationships unless you think you will actually use them for something.

Reply:
Great point! Committed.